src/test/java/redis/clients/jedis/modules/search/SearchTest.java

```diff
@@ -492,7 +623,7 @@ public void testQueryParams() {
     client.hset("2", "numval", "2");
     client.hset("3", "numval", "3");
 
-    Query query =  new Query("@numval:[$min $max]").addParam("min", 1).addParam("max", 2);
+    Query query =  new Query("@numval:[$min $max]").addParam("min", 1).addParam("max", 2).dialect(2);
```

Comment:
If this is not necessary to pass the test, please remove this change.
If you want to have this test as well, just add a new one.